python/py-imutils/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-imutils
+version             0.5.4
+platforms           darwin
+supported_archs     noarch
+license             Apache-2
+maintainers         nomaintainer
+
+description         Python image processing functions with OpenCV
+
+long_description    A series of convenience functions to make basic \
+                    image processing functions such as translation, \
+                    rotation, resizing, skeletonization, displaying \
+                    Matplotlib images, sorting contours, detecting \
+                    edges, and much more easier with OpenCV and both \
+                    Python 2.7 and Python 3.
+
+homepage            https://github.com/jrosebr1/imutils
+master_sites        pypi:i/imutils
+distname            imutils-${version}
```

Comment:
you can remove `master_sites` and `distname`, these are the defaults set by the python PG

Reply:
Thanks, done.